src/Http/Retry.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace ChartMogul\Http;
+
+use Psr\Http\Message\ResponseInterface;
+use STS\Backoff\Backoff;
+
+class ExponentialStrategy extends \STS\Backoff\Strategies\ExponentialStrategy
+{
+    // min wait time
+    protected $base = 1000;
+}
+
+class Retry
+{
+
+    private $retries;
+
+    public function __construct($retries)
+    {
+        $this->retries = $retries;
+    }
+
+    private function retryHTTPStatus($status)
+    {
+        return $status == 429 || ($status >= 500 && $status < 600);
+    }
+
+    protected function shouldRetry($attempt, $maxAttempts, ResponseInterface $response = null, \Exception $ex = null)
+    {
+        if ($attempt >= $maxAttempts && ! is_null($ex)) {
```

Comment:
Just some formatting here: ! is not near is_null

Reply:
hmm, wonder why `php_codesniffer` didn't fix it :thinking: